src/article/components/SubjectEditor.js

```diff
@@ -0,0 +1,57 @@
+import DropdownEditor from '../shared/DropdownEditor';
+import { PREVIEW_MODE } from '../ArticleConstants';
+
+const subjectTypes = [
+  'Biochemistry and Chemical Biology',
+  'Cancer Biology',
+  'Cell Biology',
+  'Chromosomes and Gene Expression',
+  'Computational and Systems Biology',
+  'Developmental Biology',
+  'Ecology',
+  'Epidemiology and Global Health',
+  'Evolutionary Biology',
+  'Genetics and Genomics',
+  'Human Biology and Medicine',
+  'Immunology and Inflammation',
+  'Microbiology and Infectious Disease',
+  'Neuroscience',
+  'Physics of Living Systems',
+  'Plant Biology',
+  'Stem Cells and Regenerative Medicine',
+  'Structural Biology and Molecular Biophysics',
+];
+
+const articleTypes = ['Insight', 'Research article', 'Tools and resources', 'Short report', 'Editorial'];
```

Comment:
Is this a temp solution to keep those lists hardcoded here? 
One is an article type the other one is a subject. Would be nice to have something reusable and configurable with a list of options. Appreciate it could be an impossible thing to do withing this framework

Reply:
Hmm, good point. Yes it is temporary, but I think I might actually move these to somewhere more global alongside other similar configuration items.
Dynamic configuration of the tool at run time (without needing the change the code) is a big item that we will be tackling later in the project.

Reply:
Ok, resolved in recent push